apps/cnquery/cmd/plugin.go

```diff
@@ -109,10 +109,12 @@ func (c *cnqueryPlugin) RunQuery(conf *run.RunQueryConfig, runtime *providers.Ru
 
 	for i := range assets {
 		connectAsset := assets[i]
-		if err := runtime.DetectProvider(connectAsset); err != nil {
+		// FIXME: I assume we need to transfer settings like recording, etc. to the new runtime
```

Comment:
Perhaps we need a clone function like we now also have for the conf inside k8s and was providers?